internal/resources/slo/resource_slo.go

```diff
@@ -492,16 +489,16 @@ func packSloResource(d *schema.ResourceData) gapi.Slo {
 func packQuery(tfquery map[string]interface{}) gapi.Query {
 	var query gapi.Query
 
-	querytype := tfquery["query_type"].(string)
-	// querythresholdstruct := tfquery["threshold"].([]interface{})
-	// querylabelsstruct := tfquery["group_by_labels"].([]interface{})
+	queryType := tfquery["query_type"].(string)
+	// queryThresholdStruct := tfquery["threshold"].([]interface{})
+	queryLabelsStruct := tfquery["group_by_labels"].([]interface{})
 
-	switch querytype {
+	switch queryType {
 	case "freeform":
 		freeformQuery := tfquery["freeform_query"].(string)
 		query = gapi.Query{
-			FreeformQuery: generateFreeformQuery(freeformQuery),
-			// GroupByLabels: generateQueryLabels(querylabelsstruct),
+			FreeformQuery: packFreeformQuery(freeformQuery),
+			GroupByLabels: packQueryLabels(queryLabelsStruct),
```

Comment:
I didn't think we could use GroupByLabels with the FreeFormQuery... wouldn't that imply that we'd be modifying the FreeFormQuery to add some additional aggregation / grouping?

Do you have an example where this is needed